libs/components/core/src/lib/modules/help/help-open-args.ts

```diff
@@ -3,7 +3,7 @@
  */
 export interface SkyHelpOpenArgs {
   /**
-   * A unique key that identifies the help content to display.
+   * A unique key that identifies the help content to display. If not provided, the page's default help content will be displayed.
    * */
-  helpKey: string;
+  helpKey?: string;
```

Comment:
Done. I like this idea. I originally was leaning towards keeping it the way that I had it in case we ever wanted to allow for a required argument to be provided without help key. However, we would need to make a change either way at that point - so I think this is cleaner for now.